plugin/adsk/scripts/mayaUSDRegisterStrings.mel

```diff
@@ -243,6 +243,28 @@ global proc mayaUSDRegisterStrings()
     register("kMayaRefDuplicateAsUsdDataDiv", "Stages");
     register("kMayaRefDuplicateAsUsdDataOptions", "Options...");
 
+    // All strings for the USD Global Preferences:
+    register("kUSDPreferences", "USD: USD Preferences");
+    register("KSavingUSDFiles", "Save USD Files");
+    register("kdotusdFileFormat", ".usd File Format:");
+    register("kAscii", "ASCII");
+    register("kBinary", "Binary");
+    register("kSaveLayerUsdFileFormatAnn", "Select whether the .usd file is written out in binary or ASCII. You can save a file in .usdc (binary) or .usda (ASCII) format. Manually entering a file name with an extension overrides the selection in this preference.");
```

Comment:
I suppose this comes Anisha, but to me it reads better if "You can save" is replaced with "You can also save", otherwise I feel there is a missing continuity between the two sentences.

Reply:
I assume that it did initially come from Anisha. These particular strings were just moved here from the LayerEditor string resources. So they are not new strings.